Source/Core/DolphinWX/ISOProperties/ISOProperties.cpp

```diff
@@ -231,13 +231,16 @@ void CISOProperties::CreateGUIControls()
 {
   const int space5 = FromDIP(5);
 
-  wxButton* const edit_config = new wxButton(this, ID_EDITCONFIG, _("Edit Config"));
-  edit_config->SetToolTip(_("This will let you manually edit the INI config file."));
+  wxButton* const edit_config = new wxButton(this, ID_EDITCONFIG, _("Edit User Config"));
+  edit_config->SetToolTip(
+      _("This will let you edit the user config INI file for this game. Changes should be "
+        "made to the user config and not the default config."));
 
   wxButton* const edit_default_config =
-      new wxButton(this, ID_SHOWDEFAULTCONFIG, _("Show Defaults"));
-  edit_default_config->SetToolTip(
-      _("Opens the default (read-only) configuration for this game in an external text editor."));
+      new wxButton(this, ID_SHOWDEFAULTCONFIG, _("View Default Config"));
+  edit_default_config->SetToolTip(_(
+      "This will let you view the default config INI file(s) for this game. Changes should be made "
```

Comment:
Similar complaints here. You isn't great, and configuration shouldn't be shortened in the tooltip (at least not the first statement of the word). Also something a little more verbous about why they shouldn't edit it is good too (and helps further deter editing!).

I recommend: "Displays the default configuration INI file(s) for this game. These defaults are recommended settings from the developers to avoid known issues. Changes should be made to the user config INI files only, not to default config INI files."

Since delroth is just going to remove this button eventually anyway, I don't care toooo much about this, though.

Reply:
What is the replacement to the button? Will it no longer be possible to display default INIs? That indeed wouldn't be needed if the whole content of the default INIs would be copied to the user INIs when the user decides to "restore defaults" for example.